### PR TITLE
graphics_functional: Add cases for filetransfer

### PR DESCRIPTION
**Pull request by cliping**
Signed-off-by: Liping Cheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Test result:
#avocado run --vt-type libvirt --vt-machine-type q35 graphics.functional.positive_tests.spice_only.filetransfer_enabled
JOB ID     : 229d10f8f5253f3370e92fa541d5b903ab77dbb3
JOB LOG    : /root/avocado/job-results/job-2019-11-22T04.00-229d10f/job.log
 (1/2) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.spice_only.filetransfer_enabled.default: PASS (10.60 s)
 (2/2) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.spice_only.filetransfer_enabled.spice_tls_disabled: PASS (9.23 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 21.41 s

#avocado run --vt-type libvirt --vt-machine-type q35 graphics.functional.positive_tests.spice_only.clipboard_enabled
JOB ID     : 9b73d7ff4d6a12b99611dbee603541248bd41ec7
JOB LOG    : /root/avocado/job-results/job-2019-11-22T04.03-9b73d7f/job.log
 (1/2) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.spice_only.clipboard_enabled.default: PASS (10.60 s)
 (2/2) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.spice_only.clipboard_enabled.spice_tls_disabled: PASS (10.15 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 22.36 s

#avocado run --vt-type libvirt --vt-machine-type q35 graphics.functional.positive_tests.spice_only.clipboard_disabled
JOB ID     : 398dce5f4292cde919e029661587061f55e1440b
JOB LOG    : /root/avocado/job-results/job-2019-11-22T04.04-398dce5/job.log
 (1/2) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.spice_only.clipboard_disabled.default: PASS (9.56 s)
 (2/2) type_specific.io-github-autotest-libvirt.graphics.functional.positive_tests.spice_only.clipboard_disabled.spice_tls_disabled: PASS (9.75 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 20.98 s

**Comment by cliping** [comment]
Relate patch:
https://github.com/avocado-framework/avocado-vt/pull/2346

File: libvirt/tests/src/graphics/graphics_functional.py [hunk 1]
```diff
@@ -1254,6 +1258,23 @@ def create_secret(secret_uuid, secret_password):
     return encryption_uuid
 
 
+def check_xml(params, test, graphic_type="vnc"):
```
Comment (chloerh) [inline libvirt/tests/src/graphics/graphics_functional.py]: Looks like var 'graphic_type' is useless in this function?

Comment (cliping) [reply libvirt/tests/src/graphics/graphics_functional.py]: @chloerh , removed 'graphic_type' parameter. Thanks.

File: libvirt/tests/src/graphics/graphics_functional.py [hunk 2]
```diff
@@ -1369,6 +1390,7 @@ def run(test, params, env):
             spice_opts, plaintext_channels, tls_channels = qemu_spice_options(vm)
             check_spice_result(spice_opts, plaintext_channels, tls_channels,
                                expected_result, all_ips, test)
+            check_xml(params, test)
```
Comment (chunfuwen) [inline libvirt/tests/src/graphics/graphics_functional.py]: Do you need add some precondition ,such as if filetransfer, before call check_xml?

Comment (cliping) [reply libvirt/tests/src/graphics/graphics_functional.py]: @chunfuwen , updated check_xml, thanks.

**Pull request merged**